src/lando/api/legacy/workers/landing_worker.py

```diff
@@ -129,6 +129,7 @@ def notify_user_of_landing_failure(job: LandingJob):
             job.requester_email, job.landing_job_identifier, job.error, job.id
         )
 
+    # XXX Not covered by tests
```

Comment:
Should probably file a bug for this.

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1935549